README.md

```diff
@@ -26,7 +26,7 @@ of useful vector and large number operations:
 
 It is likely sufficient to perform:
 
-    $ ./configure
+    $ ./configure  CFLAGS="-O3 -mcpu=power8"
```

Comment:
Do you need `-mcpu=power8` ?  If you're on POWER8, then you're on LE, and the default for LE is `-mcpu=power8`.

Reply:
> If you're on POWER8, then you're on LE
That's not accurate, of course.  Sorry.  Ignore me.

Reply:
Ok ;)

Yes POWER8 can be either, But POWER7 is only BE.

I did not want to get into the whole support matrix mess. Just provide an example.